lesson10/index.js

```diff
@@ -26,30 +27,33 @@ function getData() {
   });
 }
 
-async function createLists() {
+async function fetchData() {
   try {
-    const responseData = await getData();
-    const frag = document.createDocumentFragment();
-    responseData.forEach((data) => {
-      const li = document.createElement('li');
-      const anchor = document.createElement('a');
-      const img = document.createElement('img');
-
-      anchor.href = `/${data.to}`;
-      img.src = data.img;
-      img.alt = data.alt;
-
-      frag.appendChild(li).appendChild(anchor).appendChild(img);
-      anchor.insertAdjacentHTML('beforeend', data.text);
-    });
-    ul.appendChild(frag);
-  } catch (err) {
-    console.error(err);
-  }
-  finally {
+    return await getData();
+  } catch (e) {
+    console.error(e);
+    ul.textContent = "エラーが発生しました！";
```

Comment:
(nist)

I'm sorry for the correction based on the review.

If it is ul.textContent, the text will be included in the <ul> tag.
I think it's better to put text other than the ul tag!

Reply:
Thanks for the review :)
I understood that it is not a good idea to put text directly in the ul.
Fixed the code to get the div tag and insert the text inside the div when it catches an error.

Please confirm.
17b93893959ee33bfcddd8b75d2aa999182eec06